osu.Game/Beatmaps/Formats/OsuLegacyDecoder.cs

```diff
@@ -270,6 +274,24 @@ private void handleEvents(Beatmap beatmap, string line)
             }
         }
 
+        private void handleFilenames(ref string[] split)

```

Comment:
Yes it does. Without it line 290 would not have any effect outside of the method. The `ref` keyword turns it into a reference to a reference to a string array.